test/logs.js

```diff
@@ -179,6 +179,8 @@ QUnit.test( module1Test2.name, function( assert ) {
 	delete testDoneContext.duration;
 	// Delete testDoneContext.assertions so we can easily jump to next assertion
 	delete testDoneContext.assertions;
+	// Delete testDoneContext.source
```

Comment:
The code style requires a blank space above comments, and you may also include the missing blank on the other comment above, if you don't mind.